ansible/roles/search-service-deploy/tasks/main.yml

```diff
@@ -40,6 +43,11 @@
   become_user: "{{learner_user}}"
   template: src=search-service.conf.j2 dest={{ search_root }}/search-service.conf mode=755 owner={{ learner_user }} group={{ learner_user }}
 
+- name: delete existing search artifact
+  become: yes
+  become_user: "{{learner_user}}"
+  file: name={{ search_root }}/search-manager-1.0-SNAPSHOT-dist.zip state=absent
```

Comment:
@G33tha Please use a varible in defaults with the value being the file name as this is repeating multiple times in role, it would be good to update in one place in case of a change

Reply:
@keshavprasadms above changes have been done. Please review the changes.